### PR TITLE
Change template to use cookiecutter  for backstage

### PR DESCRIPTION
**Pull request by adusumillipraveen**
AB#1013

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        type: string
+        description: Name of the component, e.g. frontend
+      description:
+        title: Description
+        type: string
+        description: Description of the component
+      http_port:
+        title: Port
+        type: integer
+        default: 8080
```
Comment (timja) [inline template.yaml:36]: cookie cutter default is using 3100 why different?

File: {{cookiecutter.component_id}}/charts/{{ cookiecutter.app_full_name }}/values.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+nodejs:
+  applicationPort: {{ cookiecutter.http_port }}
+  aadIdentityName: rpe
+{% raw %}
+  ingressHost: {{ cookiecutter.app_full_name }}.{{ .Values.global.environment }}.platform.hmcts.net
```
Comment (timja) [inline {{cookiecutter.component_id}}/charts/{{ cookiecutter.app_full_name }}/values.yaml:5]: does this work with raw?

File: {{cookiecutter.component_id}}/docker-compose.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@ version: '2'
 
 services:
   # TODO: change this name to match the name of your app
```
Comment (timja) [inline {{cookiecutter.component_id}}/docker-compose.yml:4]:
```suggestion
```

**Review by timja — APPROVED** [review]
few minor comments

File: template.yaml (hunk 1 above)
Comment (adusumillipraveen) [reply template.yaml:36]: copy paste :)

File: {{cookiecutter.component_id}}/charts/{{ cookiecutter.app_full_name }}/values.yaml (hunk 2 above)
Comment (adusumillipraveen) [reply {{cookiecutter.component_id}}/charts/{{ cookiecutter.app_full_name }}/values.yaml]: fixed it

**Pull request merged**